packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/components/FEAASElementOverrides.tsx

```diff
@@ -0,0 +1,18 @@
+import Image from 'next/image';
+import * as FEAAS from '@sitecore-feaas/clientside/react';
+
+// Element implementations for Sitecore Component Builder can be overriden here
+
+const FEAASElementOverrides = (): JSX.Element => {
+  // Register next Image to be used in Component Builder.
+  // Nextjs image implementation will be used when img is rendered in component from Component Builder
+  FEAAS.setElementImplementation('img', (attributes: Record<string, string>) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { children, src, alt, ...imgAttributes } = attributes;
+    return <Image height={1920} width={1200} src={src} alt={alt} {...imgAttributes} />;
```

Comment:
if image is not whitelisted it will not be rendered. That's why we separate whitelist presets for development/editing and production like. The former is much more volatile in terms of source of images being added - so we ensure _every_ feaas image is displayed. While we expect developers to adjust the whitelisted sources for production as they see fit

Reply:
you can check presets in nextjs.config